### PR TITLE
Fix for #3655 - always print after gcode send

### PR DESCRIPTION
**Pull request by gege2b**

### CODE REVIEW

File: lib/Slic3r/GUI/Plater.pm [hunk 1]
```diff
@@ -1399,7 +1399,7 @@ sub send_gcode {
             # OctoPrint doesn't like Windows paths so we use basename()
             # Also, since we need to read from filesystem we process it through encode_path()
             file => [ $path, basename($path) ],
-            print => $self->{send_gcode_file} ? 1 : 0,
+            print => $self->{send_gcode_file_print} ? 1 : 0,
```
Comment (lordofhyphens) [inline lib/Slic3r/GUI/Plater.pm]: Basically a typo?

Comment (gege2b) [inline lib/Slic3r/GUI/Plater.pm]: basically, yeah :)

**Comment by alexrj** [comment]
Great job! :-)))))))))

File: lib/Slic3r/GUI/Plater.pm (hunk 1 above)
Comment (lordofhyphens) [inline lib/Slic3r/GUI/Plater.pm]: Yeah, I said you should put it in so your name is in the commit log for it ;) 

I would have merged it in myself if I'd had time to test it locally.

**Comment by dwillmore** [comment]
Does this make the "send to printer" button always send & print?  Because that would break my workflow.

**Comment by gege2b** [comment]
no, it just make the confirmation box act as intended

I feature request in #3657 to make the "send & print" feature really optionnal (just a checkbox and no confirmation dialog)
Still waiting for comments

**Pull request merged**